.github/workflows/scorecards.yml

```diff
@@ -67,6 +67,6 @@ jobs:
 
       # Upload the results to GitHub's code scanning dashboard.
       - name: "Upload to code-scanning"
-        uses: github/codeql-action/upload-sarif@cdcdbb579706841c47f7063dda365e292e5cad7a  # v2.13.4
+        uses: github/codeql-action/upload-sarif@9fdb3e49720b44c48891d036bb502feb25684276  # v3.25.6
```

Comment:
The current release is 2.17.3, not sure what dependabot is picking up here.

Reply:
That looks like a github tag with a bad version number. If we jump to that, I'd worry that dependabot won't pickup future releases assuming they are downgrades.